### PR TITLE
Sentry update to the code

### PR DESCRIPTION
**Pull request by ADIMANV**
# Pull Request

## Description
Minor Update to the code which adds Sentry for a better error detection
Stems from the PR: https://github.com/openclimatefix/ocf-infrastructure/issues/540

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by peterdudfield — APPROVED** [review]

File: satip/app.py [hunk 1]
```diff
@@ -16,15 +16,24 @@
...
 import pandas as pd
+import sentry_sdk
 import structlog
 
 import satip
 from satip import utils
 from satip.constants import RSS_ID, SEVIRI_ID, SEVIRI_IODC_ID
 from satip.eumetsat import EUMETSATDownloadManager
 
+sentry_sdk.set_tag("app_name", "SATIP")
```
Comment (peterdudfield) [inline satip/app.py:27]: could this be `satellite_consumer`

**Comment by peterdudfield** [comment]
Could you add SENTRY_DSN and ENVIRONMENT to https://github.com/openclimatefix/Satip/blob/main/README.md?plain=1#L96

File: satip/app.py (hunk 1 above)
Comment (ADIMANV) [reply satip/app.py]: yep, changed that

File: satip/app.py [hunk 2]
```diff
@@ -16,15 +16,21 @@
 
 import click
 import pandas as pd
-import structlog
```
Comment (peterdudfield) [inline satip/app.py:19]: put back in

**Comment by peterdudfield** [comment]
Looks good to go

**Pull request merged**